tests/integration/test_service_manager.py

```diff
@@ -71,8 +92,13 @@ def discovery_client(discovery_service_stub: FakeDiscoveryServiceStub) -> Discov
 
 
 @pytest.fixture
-def discovery_service_stub() -> FakeDiscoveryServiceStub:
-    """Create a FakeDiscoveryServiceStub."""
+def discovery_service_stub(request) -> FakeDiscoveryServiceStub:
```

Comment:
type hint can be updated accordingly.

Reply:
Removed this marker since I have changed to direct test parameterization approach.